### PR TITLE
Make the DevTools default theme configurable for g3

### PR DESCRIPTION
**Pull request by kenzieschmoll**
RELEASE_NOTE_EXCEPTION=[g3 change]

### CODE REVIEW

**Review by polina-c — APPROVED** [review]

File: packages/devtools_app/test/cpu_profiler/cpu_profiler_test.dart [hunk 1]
```diff
@@ -2,24 +2,15 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:devtools_app/devtools_app.dart';
```
Comment (bkonyi) [inline packages/devtools_app/test/cpu_profiler/cpu_profiler_test.dart:5]: Was this import intentional?

File: packages/devtools_app/test/inspector/inspector_integration_test.dart [hunk 2]
```diff
@@ -2,12 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:devtools_app/src/screens/debugger/breakpoint_manager.dart';
-import 'package:devtools_app/src/screens/inspector/inspector_screen.dart';
-import 'package:devtools_app/src/shared/config_specific/ide_theme/ide_theme.dart';
-import 'package:devtools_app/src/shared/diagnostics/inspector_service.dart';
-import 'package:devtools_app/src/shared/globals.dart';
-import 'package:devtools_app/src/shared/notifications.dart';
+import 'package:devtools_app/devtools_app.dart';
```
Comment (bkonyi) [inline packages/devtools_app/test/inspector/inspector_integration_test.dart:5]: Ditto

File: packages/devtools_app/test/memory/memory_screen_test.dart [hunk 3]
```diff
@@ -2,20 +2,13 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:devtools_app/src/screens/memory/memory_controller.dart';
-import 'package:devtools_app/src/screens/memory/memory_screen.dart';
+import 'package:devtools_app/devtools_app.dart';
```
Comment (bkonyi) [inline packages/devtools_app/test/memory/memory_screen_test.dart:5]: Ditto

File: packages/devtools_app/test/test_infra/scenes/memory/default.dart [hunk 4]
```diff
@@ -2,18 +2,12 @@
 // Use of this source code is governed by a BSD-style license that can be found
 // in the LICENSE file.
 
-import 'package:devtools_app/src/screens/memory/memory_controller.dart';
-import 'package:devtools_app/src/screens/memory/memory_screen.dart';
+import 'package:devtools_app/devtools_app.dart';
```
Comment (bkonyi) [inline packages/devtools_app/test/test_infra/scenes/memory/default.dart:5]: Ditto

File: packages/devtools_app/test/cpu_profiler/cpu_profiler_test.dart (hunk 1 above)
Comment (kenzieschmoll) [reply packages/devtools_app/test/cpu_profiler/cpu_profiler_test.dart:5]: yeah, devtools_app.dart import is okay for tests, just not in src/

**Pull request merged**